app/presenters/link_check_csv_presenter.rb

```diff
@@ -0,0 +1,21 @@
+require 'csv'
+
+class LinkCheckCSVPresenter
+  def self.homepage_links_status_csv
+    CSV.generate do |csv|
+      csv << %w(status count)
+      LocalAuthority.group(:status).count.each do |key, value|
+        csv << [key || "nil", value]
+      end
+    end
+  end
+
+  def self.links_status_csv
+    CSV.generate do |csv|
+      csv << %w(status count)
+      Link.group(:status).count.each do |key, value|
```

Comment:
I agree although it does slightly beg the question why we hold on to old links for services that we don't care about and don't check.  Should we either check them or get rid of them?  (That's not really to do with this PR though, sorry)